### PR TITLE
AddViews

### PR DESCRIPTION
**Pull request by Ludick-Dev**

### CODE REVIEW

File: CallCenter/Views/Admin/AdminEmployees.cshtml [hunk 1]
```diff
@@ -9,6 +9,118 @@
...
+                            <th class="sortable" data-sort="serviceTypes">Service Types</th>
+                            <th class="sortable" data-sort="certificationLevel">
+                                Certification Level
+                            </th>
+                            <th class="sortable" data-sort="availability">Availability</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td>43</td>
```
Comment (Albrecht-Ohsiek) [inline CallCenter/Views/Admin/AdminEmployees.cshtml:55]: I assume you will change this to be dynamic with the database content

File: CallCenter/Views/Admin/AdminRequestLogs.cshtml [hunk 2]
```diff
@@ -2,9 +2,113 @@
...
+                            <th data-sort="callDuration">Call Duration</th>
+                            <th data-sort="status">Status</th>
+                            <th data-sort="lastCall">Last Call</th>
+                            <th data-sort="priority">Priority</th>
+                            <th data-sort="clientId">Client ID</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td>45</td>
```
Comment (Albrecht-Ohsiek) [inline CallCenter/Views/Admin/AdminRequestLogs.cshtml:46]: Same as above

File: CallCenter/Views/CallCenter/CallCenter.cshtml [hunk 3]
```diff
@@ -57,7 +57,132 @@
...
+            </div>
 
+            <div class="history">
+                <table>
+                    <thead>
+                        <h3>Caller History:</h3>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td>data1</td>
```
Comment (Albrecht-Ohsiek) [inline CallCenter/Views/CallCenter/CallCenter.cshtml:84]: same as above

File: CallCenter/Views/ClientMaintenance/ClientMaintenance.cshtml [hunk 4]
```diff
@@ -3,8 +3,125 @@
...
+                    </div>
+                </div>
+            </div>
+
+            <hr />
 
+            <table>
+                <thead>
+                    <tr>
+                        <th></th>
```
Comment (Albrecht-Ohsiek) [inline CallCenter/Views/ClientMaintenance/ClientMaintenance.cshtml:107]: Same as above

File: CallCenter/Views/ServiceRequests/ServiceRequests.cshtml [hunk 5]
```diff
@@ -2,9 +2,197 @@
...
+                            <th onclick="sortTable(1)">Request ID</th>
+                            <th onclick="sortTable(2)">Client Name</th>
+                            <th onclick="sortTable(3)">Status</th>
+                            <th onclick="sortTable(4)">Color</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <!-- Populate this section dynamically with your data -->
+                        <tr>
+                            <td>High</td>
```
Comment (Albrecht-Ohsiek) [inline CallCenter/Views/ServiceRequests/ServiceRequests.cshtml:48]: Same as above

**Review by Albrecht-Ohsiek — APPROVED** [review]
See comments. Once comments are addressed and resolved you can merge in (Merge commit or squash and merge is fine) . In future make the PR's to develpment first and ensure they are up to date with the branch you are going to merging it into.

File: CallCenter/Views/Admin/AdminEmployees.cshtml (hunk 1 above)
Comment (Ludick-Dev) [reply CallCenter/Views/Admin/AdminEmployees.cshtml:55]: For now it's all static because it was done in a separate code space

**Pull request merged**